quick/commands/streams_app.py

```diff
@@ -58,6 +61,12 @@ def add_args(self, parser: ArgumentParser, required: ArgumentGroup, optional: Ar
             help="Docker image tag",
             required=True,
         )
+        optional.add_argument(
+            "--image-pull-secret",
+            metavar="IMAGE_PULL_SECRET",
+            type=str,
+            help="A secret for pulling an image from a private registry",

```

Comment:
In which format?

Reply:
I guess, just a string, as it is later passed to the backend, and then put into a .yml